### PR TITLE
Expand account settings page to add account info and preferences

### PR DESCRIPTION
**Pull request by YaManicKill**
Initial PR for editing account info, etc.

I've expanded the page a little bit. I've also added a section to change your email address (and any other future account info we have), but I've disabled it until we figure out the emailing thing.

### CODE REVIEW

File: client/account/info.ctrl.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = function (io, $mdToast, errorHandler) {
```
Comment (not-an-aardvark) [inline client/account/info.ctrl.js]: Could you change this to use classes/prototype functions now that the other controllers also do that?

File: client/account/password.ctrl.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module.exports = function (io, $mdToast, errorHandler) {
```
Comment (not-an-aardvark) [inline client/account/password.ctrl.js]: Same here

File: client/account/prefs.ctrl.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+module.exports = function (io, $mdToast, errorHandler) {
```
Comment (not-an-aardvark) [inline client/account/prefs.ctrl.js]: and here

**Comment by not-an-aardvark** [comment]
Now that preferences are going to be changeable, we might want to fix [this](https://github.com/porygonco/porybox/blob/8b448b0c1fa7a0372f4c66bf9386d2aeac57fa07/client/add/box.ctrl.js#L12) so that it actually depends on the preference.

**Comment by YaManicKill** [comment]
@not-an-aardvark Done

File: client/account/info.ctrl.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module.exports = class Info {
+  constructor (io, $mdToast, errorHandler) {
+    this.email = this.email || '';
+    this.io = io;
+    this.$mdToast = $mdToast;
+    this.errorHandler = errorHandler;
+  }
+  saveInfo () {
+    console.log(this.email);
```
Comment (not-an-aardvark) [inline client/account/info.ctrl.js]: This console.log statement isn't supposed to be here, right?

Comment (YaManicKill) [inline client/account/info.ctrl.js]: Well, this doesn't do anything yet.

File: client/account/account.module.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    $routeProvider.when('/account', {
+      templateUrl: '/account/account.view.html',
+      controller: [accountCtrl],
+      controllerAs: 'account'
+    });
+  }])
+  .component('editPassword', {
+    templateUrl: 'account/password.view.html',
+    controller: ['io', '$mdToast', 'errorHandler', passwordCtrl],
+    controllerAs: 'password'
```
Comment (not-an-aardvark) [inline client/account/account.module.js]: Make sure to edit the `profile.something` statements to `password.something` in `client/account/password.view.html`. At the moment the form for changing your password doesn't work.

Comment (YaManicKill) [inline client/account/account.module.js]: Sigh. I was half asleep when I did the last refactor, I think. Never thought to re-test this.

**Pull request merged**